source/use-cases/sensitive-data-encryption.txt

```diff
@@ -60,12 +60,13 @@ B. Define a JSON Schema
 MongoDB uses JSON Schema to validate data that is written to a collection.
 Within the context of Field Level Encryption, JSONSchema is used to
 specify several configuration details.
-For instance, you can specify the form of encryption you want to use  -
-deterministic or random.
-You can also specify the BSON Type of the field you are encrypting. In
-addition, you can 
-specify the key you want to use. This is particularly important, because
-it means that you can use different keys for different fields.
+
+* Drivers can automatically encrypt and decrypt fields
+* To do so, you need to specify two things in JSON Schema for each encrypted field:
+    a) The encryption type (deterministic or random)
+    b) The data key
+* You can use a single data key for all encrypted fields or you can use individual data keys that are specific to each field.
```

Comment:
FYI: I meant these bullet points as a reference for the structure of the previous paragraph moreso than a verbatim bulleted list

```rst
MongoDB natively supports data validation by allowing you to specify
a JSON Schema for each collection. In addition to standard schema
validation, you can specify field-level encryption details in a
collection's JSON Schema to allow MongoDB drivers to automatically
encrypt and decrypt fields.

To configure encryption for a field in a JSON Schema, you must specify
the following in the `encrypt` field of the field's schema:

- The encryption algorithm (i.e. deterministic or random)
- The data key used to encrypt and decrypt the field
- The BSON type of the field

.. code-block:: json
   
   "<field name>": {
     "encrypt": {
       "bsonType": "<BSON Type>",
       "algorithm": "<Encryption Algorithm>",
       "keyId": [
         {
           "$binary": {
             "base64": <keyId>,
             "subType": "04"
           }
         }
       ],
     }
   }
```

Reply:
I actually like the a) ... b) ... format. Also, like you said, since bsonType isn't relevant to jsonSchema I've abstracted it for this piece of doc. I did take your suggestion on having that code block implemented.

This is implemented in the latest commit.

Reply:
Do you mean, the first bullet point as in "The encryption type (deterministic or random)" is re-iterating? If we're getting rid of that we might as well get rid of the data-key one